src/components/DisplayCard/DisplayCard.tsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  Box,
+  Divider,
+  HStack,
+  StylesProvider,
+  Text,
+  useMultiStyleConfig,
+  useStyles,
+  VStack,
+} from "@chakra-ui/react"
+import type {
+  BoxProps,
+  omitThemingProps,
+  StackProps,
+  TextProps,
+} from "@chakra-ui/react"
+import { createContext } from "react"
+
+import {
+  DisplayCardVariant,
+  DISPLAY_CARD_THEME_KEY,
+} from "theme/components/DisplayCard"
+
+interface DisplayCardProps extends StackProps {
+  variant?: DisplayCardVariant
+}
+
+interface DisplayCardHeaderProps extends BoxProps {
+  button?: JSX.Element
+}
+
+interface DisplayCardTitleProps extends TextProps {
+  icon?: JSX.Element
+  hasUnderline?: boolean
+}
+
+interface DisplayCardContextProps extends ReturnType<typeof omitThemingProps> {
+  variant: DisplayCardVariant
```

Comment:
I'm not sure how to adjust this. The variant is used in the DisplayCard context provider so that we can specify the variant of DisplayCard to use. Do you have an example for this change?

Reply:
hmm i think you might be misunderstanding either how `chakra` applies styling or my comment above! chakra extracts the styling based on the object and the specified props, given the styling in `theme/<xxx>`. this is then placed into our `StylesProvider` which provides the styles to its children. 

as we don't use `variant` anymore than this (at top level) to generate the styling, it seems to be that we can eliminate this component totally and instead just use `StyleProvider` for this

Reply:
Oops I misunderstood how Chakra uses this, adjusted in d5b6d19.